docs/docs/localization-i18n.md

```diff
@@ -12,8 +12,16 @@ This guide is a brief look at the options that exist for enhancing your Gatsby p
 
 There are a few React i18n packages out there. Several options include [react-intl](https://github.com/yahoo/react-intl), the community [Gatsby plugin](https://www.npmjs.com/package/gatsby-plugin-i18n) and [react-i18next](https://github.com/i18next/react-i18next/). There are several factors to consider when choosing a package: Do you already use a similar package in another project? How well does the package meet the needs of your users? Are you or your team already familiar with a certain package? Is the package well documented and maintained?
 
+<Accordion>
```

Comment:
This usage works better than the internals terminology doc in my opinion, as the interface presents three package options as choices the user has to make. Ultimately, Gatsby should be more decisive about which direction we recommend for localization – but we won't have that solidified quite yet until the [Internalization project](https://github.com/gatsbyjs/rfcs/pull/42) is farther along.